### ISSUE TITLE
Warning with Gradle 6.0.1

### ISSUE THREAD

**Issue by bentoi** [zeroc-ice/ice-builder-gradle#9]
I'm getting this warning when using Gradle 6.0.1 and `--warning-mode all`:
```
> Task :demoIceAsyncInvocation:compileSlice
Property 'timestamps' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
```

**Comment by pepone** [comment]
See https://docs.gradle.org/6.3/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.

**Comment by pepone** [comment]
Fixed with #11
